tsl/test/sql/continuous_aggs_ddl.sql

```diff
@@ -550,8 +551,24 @@ SELECT user_view,
  WHERE mat_table::text = table_name
    AND user_view::text LIKE 'whatever_view%';
 
+-- Checking that the access method for the materialized hypertable is
+-- set to the correct access method.
+CREATE MATERIALIZED VIEW access_view_1 USING heap2
```

Comment:
Looks like test output file is not updated for this commit :)

Reply:
Fixed.